meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -122,20 +122,10 @@ export class MongoEntityConverter {
 
 	public convertToMongoRundown(rundown: Rundown): MongoRundown {
 		return {
-			isActive: rundown.isActive(),
-			externalId: '', // Todo: figure out where the value for this attribute is
-			metaData: { rank: 0 }, // Todo: figure out where the value for this attribute is
-			modified: rundown.getLastTimeModified(),
-			notes: [], // Todo: figure out where the value for this attribute is
-			organizationId: '', // Todo: figure out where the value for this attribute is
-			playlistExternalId: '', // Todo: figure out where the value for this attribute is
-			showStyleBaseId: '', // Todo: figure out where the value for this attribute is
-			showStyleVariantId: '', // Todo: figure out where the value for this attribute is
-			studioId: '', // Todo: figure out where the value for this attribute is
-			timing: { expectedDuration: 0, expectedEnd: 0, expectedStart: 0, type: '' }, // Todo: figure out where the value for this attribute is
 			_id: rundown.id,
 			name: rundown.name,
-		}
+			isActive: rundown.isActive(),
+		} as MongoRundown
```

Comment:
I am not a fan of us casting constructs this way. If we are not going to use/care about the attributes that were removed, then instead we should remove them from the `MongoX` types.

Reply:
We can't remove them from the `MongoX` types, we still need to fetch that data from the database.

We don't want to save all these attributes to the database yet, because at this stage we still haven't taken over the responsibility of ingest.
We do however need to save some attributes to the database, but these are "new" attributes on the collection that only we are using. (Currently that is isActive on Rundown, Segment and Part and isNext on Segment and Part)